dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -189,7 +189,10 @@ private static Path generate(Bom bom) throws IOException, TemplateException, URI
     // When checking a BOM, entry point classes are the ones in the artifacts listed in the BOM
     List<ClassPathEntry> artifactJarsInBom = classpath.subList(0, managedDependencies.size());
     ImmutableSet<ClassPathEntry> entryPoints = ImmutableSet.copyOf(artifactJarsInBom);
-
+    for (ClassPathEntry entry : entryPoints) {
+      entry.loadClassFileNames();
+    }
+    
```

Comment:
I feel this piece of code is wrong.  It is as if `classPathResult.getClassPath()` has a contract that its caller needs to call `loadClassFileNames()` for each elements.

Reply:
I don't like this either I haven't found something I like better yet though. I was hoping you might have an idea.